karma.conf.js

```diff
@@ -53,7 +53,8 @@ module.exports = function(config) {
       ...filesConfig.coreBundle("src"),
       ...filesConfig.lobBundle("src"),
 
-      "src/js/modules/i18n/regional/infragistics.ui.regional-i18n.js",
+      "src/js/modules/i18n/regional/infragistics.ui.regional-ja.js",
+      "src/js/modules/i18n/regional/infragistics.ui.regional-de.js",
```

Comment:
Looking at the results, despite the built-in defaults it seems most tests rely on English file at the end too.